src/com/company/view/ConsoleReader.java

```diff
@@ -0,0 +1,23 @@
+package com.company.view;
+
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+/**
+ * Created by yulia on 13.10.16.
+ */
+public class ConsoleReader implements ConsoleManager {
```

Comment:
It's using Scanner to read console. So maybe better name is **ScannerConsoleReader**